qiskit/circuit/quantumcircuit.py

```diff
@@ -1165,6 +1166,20 @@ def append(
             if is_parameter:
                 instruction = copy.deepcopy(instruction)
 
+        # Deprecate the passing multiple qargs for contolled gates
+        if len(qargs) == 2:  # check if singly-controlled gate
+            # check for multiple qubits
+            if any(isinstance(qarg, list) and len(qarg) > 1 for qarg in qargs):
```

Comment:
`qc.cx((0, 1), 2)` won't trigger this warning, but will still have the broadcasting behaviour.

Reply:
good catch 👍🏻 fixed in `0eb0a6574`